### PR TITLE
Added website builds to MultiPackageListing.cs

### PR DESCRIPTION
**Pull request by orels1**
The MultiPackageListing now runs a website build using Scriban.

It uses index.html, app.js and styles.css and injects the data into them on build.

This requires listing repo to have those files, which are handled in its own PR to the template-package-listing repository

### CODE REVIEW

File: PackageBuilder/MultiPackageListing.cs [hunk 1]
```diff
@@ -98,6 +101,49 @@ partial class Build
                 };
                 string savePath = ListPublishDirectory / PackageListingPublishFilename;
                 repoList.Save(savePath);
+
+                var indexReadPath = PackageListingSourcePath.Parent / "Website" / WebPageIndexFilename;
+                var appReadPath = PackageListingSourcePath.Parent / "Website" / WebPageAppFilename;
+                var stylesReadPath = PackageListingSourcePath.Parent / "Website" / WebPageStylesFilename;
```
Comment (momo-the-monster) [inline PackageBuilder/MultiPackageListing.cs]:
```suggestion
                AbsolutePath WebPageSourcePath = PackageListingSourcePath.Parent / "Website";
                var indexReadPath = WebPageSourcePath / WebPageIndexFilename;
                var appReadPath = WebPageSourcePath / WebPageAppFilename;
                var stylesReadPath = WebPageSourcePath / WebPageStylesFilename;
```
This is a good candidate for introducing a reusable absolute path for readability and maintainability, I think. Caveat - I wrote this directly on GitHub, code not tested.

**Review by momo-the-monster — CHANGES_REQUESTED** [review]
The way you're checking the package dependencies might not always work - something could depend on ClientSim and not Worlds directly, for example. 

However, I've just added [two new functions](https://github.com/vrchat/vrc-package-manager/blob/a63ce6138bff7ba92034e31daac6555cabbb212c/core/vpm-core-lib/Types/PackageTypes/VRCPackageManifest.cs#L145-L161) which should help with this. I can merge them in tomorrow and add the updated library to this repo so we can use them.

(relatedly - I did make an issue to switch this repo to using a nuget package, once we publish one for vpm-core-lib: #1 )

File: PackageBuilder/MultiPackageListing.cs (hunk 1 above)
Comment (orels1) [reply PackageBuilder/MultiPackageListing.cs]: Agreed, will do and test

**Comment by orels1** [comment]
@momo-the-monster moved the variable and tested it, seems like it all works fine. I'll wait for you to upgrade the vpm core

**Review by momo-the-monster — APPROVED** [review]
Added updated vpm-core-lib and a function to return a string type based on the full dependency graph of the target package.

**Comment by orels1** [comment]
Thank you!

**Pull request merged**